ci/ray_ci/test_tester_container.py

```diff
@@ -27,24 +27,28 @@ def _mock_check_output(input: List[str]) -> None:
         assert "/bin/bash -ice run command" in input_str
 
     with mock.patch("subprocess.check_output", side_effect=_mock_check_output):
-        run_script_in_docker("run command", "team")
+        container = TesterContainer("team")
+        container.run_script("run command")
 
 
 def test_run_tests() -> None:
-    def _mock_run_tests_in_docker(test_targets: List[str], team: str) -> MockPopen:
+    def _mock_run_tests_in_docker(test_targets: List[str]) -> MockPopen:
         return MockPopen(test_targets)
 
     def _mock_shard_tests(tests: List[str], workers: int, worker_id: int) -> List[str]:
         return chunk_into_n(tests, workers)[worker_id]
 
     with mock.patch(
-        "ci.ray_ci.container._run_tests_in_docker",
+        "ci.ray_ci.tester_container.TesterContainer._run_tests_in_docker",
```

Comment:
when this thing stabilizes, I would like to do some refactor to remove all these mocks..